whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/sets/ASNameHierarchicalValidator.java

```diff
@@ -0,0 +1,41 @@
+package net.ripe.db.whois.update.handler.validator.sets;
+
+import com.google.common.collect.ImmutableList;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.update.authentication.Principal;
+import net.ripe.db.whois.update.domain.Action;
+import net.ripe.db.whois.update.domain.PreparedUpdate;
+import net.ripe.db.whois.update.domain.UpdateContext;
+import net.ripe.db.whois.update.domain.UpdateMessages;
+import net.ripe.db.whois.update.handler.validator.BusinessRuleValidator;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ASNameHierarchicalValidator implements BusinessRuleValidator {
+
+    private static final ImmutableList<Action> ACTIONS = ImmutableList.of(Action.CREATE);
+    private static final ImmutableList<ObjectType> TYPES = ImmutableList.of(ObjectType.AS_SET);
+
+    @Override
+    public void validate(final PreparedUpdate update, final UpdateContext updateContext) {
+        if (updateContext.getSubject(update).hasPrincipal(Principal.OVERRIDE_MAINTAINER)) {
+            return;
+        }
+
+        if(!StringUtils.contains(update.getUpdatedObject().getKey(),":")) {
```

Comment:
Use `update.getUpdatedObject().getKey().contains(":")` (*inverse* condition and no need for StringUtils.contains())

Reply:
StringUtils check for null as well, although at this npoint it can never be null